### ISSUE TITLE
[CI] Failure Unable to find precompiled view file (Microsoft.AspNetCore.Identity)

### ISSUE THREAD

**Issue by benaadams** [aspnet/AspNetCore#6389]
Looks so close to clean CI...
```
test\Identity.FunctionalTests\Microsoft.AspNetCore.Identity.FunctionalTests.csproj(42,5): 
 error : Unable to find precompiled view file bin\Release\netcoreapp3.0\Microsoft.AspNetCore.Identity.UI.Views.V3.dll
2019-01-04T02:12:47.8853580Z D:\a\.dotnet\buildtools\korebuild\3.0.0-build-20181214.1\modules\vstest\module.targets(102,5):
 error : Test group UnitTests/netcoreapp3.0 failed with exit code '1'. [D:\a\.dotnet\buildtools\korebuild\3.0.0-build-20181214.1\KoreBuild.proj]
 
```

**Comment by Eilon** [comment]
cc @HaoK @pranavkm who are looking into this

**Comment by pranavkm** [comment]
@natemcmaster has a PR for 2.2 - https://github.com/aspnet/AspNetCore/pull/6369. Perhaps we need to merge that in to master.

**Comment by natemcmaster** [comment]
I believe this should have been resolved by #6369 for 2.2, merged to master in https://github.com/aspnet/AspNetCore/pull/6405. We can re-open if it is still happening.
